src/boot/ksdk1.1.0/devAMG8834.h

```diff
@@ -35,7 +35,9 @@
 	POSSIBILITY OF SUCH DAMAGE.
 */
 
-
+#ifndef warp_devAMG8834_ENABLE
```

Comment:
Please see the coding conventions at https://github.com/physical-computation/Warp-firmware/blob/master/doc/README-CodingConventions.md

Reply:
is "kWarp_devAMG8834_ENABLE" ok? or is "kWarpDevAMG8834Enable" better?

Reply:
Ive renamed all the #defines to "kWarp_devAMG8834_ENABLE" if you prefer otherwise, I'll change it as required